src/twisted/python/reflect.py

```diff
@@ -134,7 +134,8 @@ def accumulateMethods(obj, dict, prefix='', curClass=None):
     if not curClass:
         curClass = obj.__class__
     for base in curClass.__bases__:
-        accumulateMethods(obj, dict, prefix, base)
+        if base is not object:
```

Comment:
I don't understand this change.  Can you add a comment here as to why this change is correct?
Does this change work on Cpython 2.7 and Cpython 3.x?